docs/APIReference-Data-Conversion.md

```diff
@@ -49,11 +49,16 @@ other usage within an application.
 ### convertFromHTML
 
 ```
-convertFromHTML(
-  html: string,
-  DOMBuilder: Function = getSafeBodyFromHTML,
-  blockRenderMap?: DraftBlockRenderMap = DefaultDraftBlockRenderMap
-): ?Array<ContentBlock>
+const sampleMarkup =
+  '<b>Bold text</b>, <i>Italic text</i><br/ ><br />' +
+  '<a href="http://www.facebook.com">Example link</a><br /><br/ >' +
+  '<img src="image.png" height="112" width="200" />';
+
+const blocksFromHTML = convertFromHTML(sampleMarkup);
+const state = ContentState.createFromBlockArray(blocksFromHTML);
```

Comment:
small nit, but since you're citing https://github.com/facebook/draft-js/blob/8ac72f723fb2d9102db833a9b060dfd66df65652/examples/convertFromHTML/convert.html#L61-L65 below, could you match it here to avoid confusion?

```
const blocksFromHTML = convertFromHTML(sampleMarkup);
const state = ContentState.createFromBlockArray(
  blocksFromHTML.contentBlocks,
  blocksFromHTML.entityMap,
);
```